### PR TITLE
Feat/table view for issues

### PR DESCRIPTION
**Pull request by YehualashetGit**
* create data table ui component
    * create re usable table component and used them for Data table UI component
    * refactor team members componen so that it can show card and table component
    * add a swtich button to change issue view

https://www.veed.io/view/f1ebbac3-64a1-4423-8e96-05685b3ea2b6?panel=share

### CODE REVIEW

**Comment by badalkhatri0924** [comment]
image



@YehualashetGit fix build issues.

**Comment by badalkhatri0924** [comment]
@YehualashetGit why needed to force push?

**Comment by YehualashetGit** [comment]
> @YehualashetGit why needed to force push?

It doesn't allow me to push after  merging the conflict

**Comment by badalkhatri0924** [comment]
> > @YehualashetGit why needed to force push?
> 
> It doesn't allow me to push after merging the conflict

@evereq what do you suggest here?

**Comment by badalkhatri0924** [comment]
image
Also @YehualashetGit it is not loading data for me.

**Comment by YehualashetGit** [comment]
> image Also @YehualashetGit it is not loading data for me.

wait let me finalize it ,it's still under Draft PR, I will make it ready for return teamMembersView;
 once I'm done, here I was doing the filter by   user id ,

File: apps/web/app/constants.ts [hunk 1]
```diff
@@ -116,3 +116,9 @@ export const APPLICATION_LANGUAGES_CODE = [
 	'ru',
 	'es'
 ];
+
+export enum KanbanView {
```
Comment (evereq) [inline apps/web/app/constants.ts:120]: Maybe let's rename this to "IssuesView"?

File: apps/web/lib/features/team/invite/user-invite-card.tsx [hunk 2]
```diff
@@ -225,9 +225,9 @@ export function InviteUserTeamCard({
 					<DraggerIcon className="fill-[#CCCCCC] dark:fill-[#4F5662]" />
 				</div>
 
-				<div className="absolute opacity-40 right-2">
+				{/* <div className="absolute opacity-40 right-2">
```
Comment (evereq) [inline apps/web/lib/features/team/invite/user-invite-card.tsx:228]: @YehualashetGit why this got commented out? Not needed anymore?

Comment (YehualashetGit) [reply apps/web/lib/features/team/invite/user-invite-card.tsx:228]: yes we don't need this code s actually,  basically the invite row won't have such menu , may be i will remove it

File: apps/web/app/constants.ts (hunk 1 above)
Comment (YehualashetGit) [reply apps/web/app/constants.ts:120]: sure I will rename that

**Comment by evereq** [comment]
@YehualashetGit see my comments and let's fix this quickly so we can merge as soon as possible, thanks!
Also, most probably we will have to make more improvements to the table view later. Honestly the idea of table view for "Issues" was different - it should be that "Issue title/description" would go as the first column, etc, but what you essentially did is exactly the same view we had for "horizontal cards" but now implemented also as a table (more condensed) view and I LIKE that. 

So we will merge this and you will work to make 3rd view for the same data (block view), but essentially we will need to also work on a real "issues" page later. Let's discuss that later in Slack, for now let's finish this PR merge it, and create a "block" view next (from our Figma):

image

File: apps/web/app/constants.ts (hunk 1 above)
Comment (evereq) [reply apps/web/app/constants.ts:120]: Btw, I think values should be:
CARDS, TABLE, and BLOCKS, please adjust @YehualashetGit  :)

Comment (YehualashetGit) [reply apps/web/app/constants.ts:120]: sure

**Comment by badalkhatri0924** [comment]
image

@YehualashetGit there are some UI issues

File: apps/web/lib/features/team/invite/user-invite-card.tsx (hunk 2 above)
Comment (evereq) [reply apps/web/lib/features/team/invite/user-invite-card.tsx]: Yes, makes sense @YehualashetGit

**Comment by badalkhatri0924** [comment]
image

@YehualashetGit UI issue.

File: apps/web/lib/features/team/invite/user-invite-card.tsx (hunk 2 above)
Comment (evereq) [reply apps/web/lib/features/team/invite/user-invite-card.tsx]: @YehualashetGit On the other hand, idea was to make that Invite row to look similar to other rows... Iike it's still "empty" but when you invite someone that person will be shown there with task and other info... So I would prefer you to restore it back :)))

Also in the future when you removing something (even if it looks like not used), please state it in the PR explicitly - we should be very careful with remove things if we not sure about initial idea why it was added etc.

Comment (YehualashetGit) [reply apps/web/lib/features/team/invite/user-invite-card.tsx]: okay understood , the main reason why I removed it was , there was no real data to display and they are just a place holder , that's why , but will reverted it back

**Comment by evereq** [comment]
@YehualashetGit you working on fixes for those issues right?

File: apps/web/lib/features/team/invite/user-invite-card.tsx (hunk 2 above)
Comment (YehualashetGit) [reply apps/web/lib/features/team/invite/user-invite-card.tsx]: @evereq  is this a temporary or it only be visible on Table view cause ,it'll now  make me to add more logic on TeamMeberTable View component to the invite row with separate actions

**Comment by YehualashetGit** [comment]
> @YehualashetGit you working on fixes for those issues right?

yes I've fixed the ui issues

**Comment by evereq** [comment]
@badalkhatri0924 can you review again please? If something still broken try to quick fix yourself and let's merge this one so I can review it online

**Review by badalkhatri0924 — APPROVED** [review]

**Comment by evereq** [comment]
@YehualashetGit I merged with "squash", so please don't use anymore that same branch, create a new one based on develop for all future work on those tasks. Thanks!

**Pull request merged**